### PR TITLE
Redis extension for PHP7

### PR DESCRIPTION
**Pull request by ruudk**
Uses the `php7` branch of phpredis

https://github.com/phpredis/phpredis/compare/develop...phpredis:php7

### CODE REVIEW

**Comment by ablyler** [comment]
@ruudk can you remove the bottle section (since this is a HEAD only formula for now)?

**Comment by ruudk** [comment]
Done! I didn't know what it was for.

File: Formula/php70-redis.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+require File.expand_path("../../Abstract/abstract-php-extension", __FILE__)
+
+class Php70Redis < AbstractPhp70Extension
+  init
+  homepage "https://github.com/phpredis/phpredis"
+  sha256 "a5882dd9b21908e123b3d5c5f72d6dc8cbbbb6a29996e568c4d18ed356c0362b"
+  head "https://github.com/phpredis/phpredis.git", :branch => "php7"
+  revision 1
+
+  depends_on "php70"
```
Comment (ablyler) [inline Formula/php70-redis.rb]: You can remove this, as the extension class will add it.

File: Formula/php70-redis.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+  def install
+
+    args = []
```
Comment (ablyler) [inline Formula/php70-redis.rb]: This can be removed.

File: Formula/php70-redis.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+    safe_phpize
+
+    system "./configure", "--prefix=#{prefix}",
+                          phpconfig,
+                          *args
```
Comment (ablyler) [inline Formula/php70-redis.rb]: This line, and the comma above can be removed

File: Formula/php70-redis.rb [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      ; To do this, uncomment and configure below
+      ;session.save_handler = redis
+      ;session.save_path = "tcp://host1:6379?weight=1, tcp://host2:6379?weight=2&timeout=2.5, tcp://host3:6379?weight=2"
+    EOS
+  end
+
+  test do
+    shell_output("php -m").include?("redis")
+  end
+end
```
Comment (ablyler) [inline Formula/php70-redis.rb]: The test block can be removed, since the class will add it.

File: Formula/php70-redis.rb [hunk 5]
```diff
@@ -0,0 +1,39 @@
+require File.expand_path("../../Abstract/abstract-php-extension", __FILE__)
+
+class Php70Redis < AbstractPhp70Extension
+  init
+  homepage "https://github.com/phpredis/phpredis"
+  sha256 "a5882dd9b21908e123b3d5c5f72d6dc8cbbbb6a29996e568c4d18ed356c0362b"
```
Comment (ablyler) [inline Formula/php70-redis.rb]: The sha256 line can be removed.

File: Formula/php70-redis.rb [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  head "https://github.com/phpredis/phpredis.git", :branch => "php7"
+  revision 1
```
Comment (ablyler) [inline Formula/php70-redis.rb]: This line can be removed.

File: Formula/php70-redis.rb [hunk 7]
```diff
@@ -0,0 +1,39 @@
+require File.expand_path("../../Abstract/abstract-php-extension", __FILE__)
+
+class Php70Redis < AbstractPhp70Extension
+  init
```
Comment (ablyler) [inline Formula/php70-redis.rb]: Add a desc line after the init line:
```
desc "PHP extension for Redis"
```

Comment (ruudk) [inline Formula/php70-redis.rb]: Done!

**Comment by ruudk** [comment]
Thanks for your comments, All done :)

**Comment by ablyler** [comment]
Merged, thanks!

**Pull request closed without merging**